Documentation/migration-guide.md

```diff
@@ -167,6 +167,14 @@ tracker.trackSessionStart(info: mediaObject, metadata: videoMetadata)
 ```
 ------
 
+### Downloaded Content Tracking
```

Comment:
nit: Can we use simple statement casing: Downloaded content tracking, Granular ad tracking? 
Also looks like these two sections are not included in the quick links